ext/crypto/00_crypto.js

```diff
@@ -978,6 +978,13 @@
         context: "Argument 2",
       });
 
+      if (!key.extractable) {
```

Comment:
```suggestion
      if (key.extractable === false) {
```

Also, this step comes after the `switch`. See https://w3c.github.io/webcrypto/#SubtleCrypto-method-exportKey

Reply:
Okay, so do you want me to implement it so that it is after the switch?

Reply:
yes

Reply:
I do not quite get why we do not throw early as @EduM22 did in their first approach. I can not see why this check should happen after the execution of the actual key extraction from the link you shared. Can you explain?